contents/tutorials/feature-retention.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to discover features that drive user retention
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-22
+featuredImage: ../images/tutorials/banners/feature-retention.png
+topics: ["stickiness", "dashboards", "retention"]
+---
+
+Every company wants to build a product that keeps users coming back. Returning and reoccurring users are often your best ones. They build are your user base,  customers, and advocates. Many teams focus on improving user retention metrics, like weekly active users or customer retention.
+
+This tutorial will help you discover the features that drive retention and keep users coming back. Specifically, we’ll build a dashboard to track metrics related to a feature’s retention and compare those metrics across your features.
+
+## Setting up the feature usage tracking
+
+The first step in understanding feature retention is knowing what data you have. You need data from your product’s features to know anything about their retention. There are two ways to do this:
+
+1. Add the [snippet](/docs/integrate/client/snippet-installation) or install the [JavaScript library](/docs/integrate/client/js) to enable [autocapture](/docs/integrate/client/js#autocapture). Find autocaptured events related to the feature (views, button presses, inputs). Create an [action](/manual/actions) (a combination of events) using the toolbar or under “Data Management” in the sidebar to represent that feature’s usage.
+2. Add event capture calls manually into the code that encompasses the feature usage. Use functions, methods, hooks, or other structures in your code to represent a feature’s usage.
+
+> **Note:** be sure to think about the relative effort it takes to use a feature. For example, a long form submission is more difficult than a video view. Users take an action more often when it is easier. An event that takes more work happens less, and this can correlate to lower retention.
```

Comment:
Love this little aside. Sprinkling in best practices is a great way to elevate the piece.